### ISSUE TITLE
The result cannot be repeated

### ISSUE THREAD

**Issue by Daybreak-Zheng** [RunpeiDong/ACT#2]
Could you please tell me why results are not reproduced on Modelnet40 classification and few shot classification? Are specific data enhancement or random seeding used?

**Comment by RunpeiDong** [comment]
Hi @Daybreak-Zheng,

Thanks for your interest. 
- For data augmentation on ModelNet40, we use `scale&translation` for both classification and few-shot settings. For classification, the results are conducted with the `voting` strategy.
- Sorry, the random seed was not recorded since we run experiments on temporary servers. But please note that the best classification accuracy on ModelNet40 is achieved by several runs with different seeds ignoring small deviations, and the voting strategy is used by default. We have provided a fine-tuned model on [Google Drive](https://drive.google.com/drive/folders/1omSypBHRUZvBilLP2Qv9DWRcnSj2VpNb), which should be slightly higher than 93.7 that was reported in the paper with voting strategy.
- For few-shot, the results are achieved by running on 10-folds, which should be more stable to achieve.
- Besides, can you please check the Pytorch version or other environment settings to see if they are the same? We have not tested the code with other environmental configurations.

**Comment by Daybreak-Zheng** [comment]
I am very glad to receive your reply！
Do you run the best result multiple times on each pkl and then calculate the mean and variance for the best result of 10 different pkls on a few shot? Or what other way of calculating it?
My experimental environment is the same as yours, except my machine is 3090.

**Comment by RunpeiDong** [comment]
Yes, exactly. I have not tried on 3090, the experiments are mainly conducted with RTX 2080Ti, P100 and A100 GPUs, but I think the results should not differ significantly. 

By the way, what learning rate do you use for few-shot?

**Comment by Daybreak-Zheng** [comment]
I followed your predefined configuration exactly, lr=0.001
Is there a special learning rate for a few shot classification?

**Comment by RunpeiDong** [comment]
The default LR should be `0.0005`, and remember that the data augmentation should be `scale&translation`.

**Comment by Daybreak-Zheng** [comment]
What is the learning rate used on the ModelNet40 classification task? LR=0.0005？
Besides, do you run the best result multiple times on each pkl and then calculate the mean and variance for the best result of 10 different pkls on a few shot? Or what other way of calculating it?

**Comment by RunpeiDong** [comment]
- Yes, the LR is 0.0005 for both ModelNet40 classification and few-shot. These hyper-parameter configurations can be found in the given [ModelNet40 configs](https://github.com/RunpeiDong/ACT/blob/master/cfgs/finetune_classification/full/finetune_modelnet.yaml) and [Few-shot configs](https://github.com/RunpeiDong/ACT/blob/master/cfgs/finetune_classification/few_shot/fewshot_modelnet.yaml), it may not be a good choice to change it.
- The way to calculate the few-shot results are achieved by running on each fold one time and report mean and variance.
- In order to help you, we have re-run the experiment with the same configuration of 5-way 10-shot with a new random seed `0` on RTX 2080Ti, and the results are as follows, which may be better when using other seeds:
![image](https://user-images.githubusercontent.com/37994246/228121531-3e802395-b477-4a0f-a14b-d3173519cf64.png)

**Comment by Daybreak-Zheng** [comment]
Thanks a lot !!!

**Comment by RunpeiDong** [comment]
You are welcome. Happy to help you.:)
